api/wik_extract.py

```diff
@@ -31,6 +33,16 @@ def wiktextract_word_cb(self, word, *args, **kwargs):
         self.clean_word(word)
         self.external_search.insert(word)
 
+    def update_language(self, language):
+        '''Update the language string into the equivalent language code. This
+        is language value that will go into the Elasticsearch document.
+        '''
+        if len(language) != 2 or len(language) != 3:
+            alpha_3 = LanguageCodes.string_to_alpha_3(language)
+            alpha_2 = LanguageCodes.three_to_two[alpha_3]
+            return alpha_2
```

Comment:
This is the same as `update_language_code`, so maybe your `normalize_language_code` can go into `LanguageCodes` and be used in both places.